frontend/src/app/join/page.js

```diff
@@ -1,43 +1,42 @@
-"use client";
```

Comment:
there shouldn't be any changes in this file

Reply:
I would probably just copy and paste what's currently in `main` for this file

Reply:
Changes were prettier formatting

Reply:
Can you rerun prettier?